lib/resources/payments.ts

```diff
@@ -0,0 +1,130 @@
+'use strict'
```

Comment:
Isn't this the default in ES modules?

Reply:
Had added this because of some issue I was facing. Will clean all of this up before merging to master.